src/pf-3-form-fields/form-fields.js

```diff
@@ -78,7 +78,7 @@ const FinalFormField = ({
         <FormGroup validationState={ invalid ? 'error' : null }>
           { label &&
           <Col md={ 12 } componentClass="label" className="control-label">
-            { !hideLabel && __(label) }
+            { !hideLabel && (rest.isRequired ? __(label).concat(' *') : __(label)) }
```

Comment:
For pf3 This should look like this: https://www.patternfly.org/pattern-library/forms-and-controls/field-labeling/